.github/workflows/discoverability.yml

```diff
@@ -0,0 +1,21 @@
+name: Discoverability
+
+on: pull_request
+
+jobs:
+  discoverability-action:
+    name: Discoverability action
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+
+    - name: yarn
+      run: |
```

Comment:
The `|` is used when chaining multiple commands but we are running one. I don't think we need to encourage more commands?